integration/nevermined/Web3Providers.test.ts

```diff
@@ -6,9 +6,9 @@ import { getMetadata } from '../utils'
 import { Nevermined, Account } from '../../src'
 
 import HDWalletProvider from '@truffle/hdwallet-provider'
```

Comment:
Is there a way to get rid of hdwallet-provider?

Reply:
yeah, ethers.js has it's own hdwallet provider